packages/dm-core-plugins/src/job/CronJob.tsx

```diff
@@ -73,131 +73,120 @@ export function ConfigureSchedule(props: {
   }
 
   return (
-    <div
-      style={{
-        border: '1px solid lightgray',
-        borderRadius: '.4rem',
-        padding: '1rem',
-        maxWidth: '800px',
-      }}
-    >
-      <div>
-        {isRegistered && (
-          <Typography color='primary'>The job is scheduled</Typography>
-        )}
-        <DateRangePicker
-          setDateRange={(dateRange) =>
-            setSchedule({
-              ...schedule,
-              startDate: dateRange.startDate,
-              endDate: dateRange.endDate,
-            })
-          }
-          value={{ startDate: schedule.startDate, endDate: schedule.endDate }}
-        />
-        <InputWrapper>
-          {showAdvanced ? (
-            <div className={'flex-col'}>
-              <TextField
-                style={{ maxWidth: '400px' }}
-                unit='cron'
-                id='advanced-schedule-syntax'
-                type='text'
-                defaultValue={schedule.cron}
-                onChange={(event: ChangeEvent<HTMLInputElement>) =>
-                  setSchedule({
-                    ...schedule,
-                    cron: event?.target.value,
-                  })
-                }
-                label='Enter explicit cron syntax'
-                helperText='minute hour day(month) month day(week)'
-              />
-              <small style={{ color: 'red' }}>
-                {' '}
-                Controls might not show correct values after manually entering
-                cron syntax
-              </small>
-            </div>
-          ) : (
-            <>
+    <div>
+      {isRegistered && (
+        <Typography color='primary'>The job is scheduled</Typography>
+      )}
+      <DateRangePicker
+        setDateRange={(dateRange) =>
+          setSchedule({
+            ...schedule,
+            startDate: dateRange.startDate,
+            endDate: dateRange.endDate,
+          })
+        }
+        value={{ startDate: schedule.startDate, endDate: schedule.endDate }}
+      />
+      <InputWrapper>
+        {showAdvanced ? (
+          <div className={'flex-col'}>
+            <TextField
+              style={{ maxWidth: '400px' }}
+              unit='cron'
+              id='advanced-schedule-syntax'
+              type='text'
+              defaultValue={schedule.cron}
+              onChange={(event: ChangeEvent<HTMLInputElement>) =>
+                setSchedule({
+                  ...schedule,
+                  cron: event?.target.value,
+                })
+              }
+              label='Enter explicit cron syntax'
+              helperText='minute hour day(month) month day(week)'
+            />
+            <small style={{ color: 'red' }}>
```

Comment:
color: red doesn't have enough contrast - should at least use tokens.colors.interactive.danger__text.hex

Reply:
aright, done!